WordPress/Classes/System/MySitesCoordinator+RootViewPresenter.swift

```diff
@@ -4,4 +4,116 @@ import Foundation
 /// and the app's UI is simplified.
 extension MySitesCoordinator: RootViewPresenter {
 
+    // MARK: General
+
+    var currentViewController: UIViewController? {
+        return rootViewController
+    }
+
+    func getMeScenePresenter() -> ScenePresenter {
+        meScenePresenter
+    }
+
+    func currentlySelectedScreen() -> String! {
+        return "Blog List"
+    }
+
+    func currentlyVisibleBlog() -> Blog? {
+        currentBlog
+    }
+
+    // MARK: Reader
+
+    var readerTabViewController: ReaderTabViewController? {
+        return nil
+    }
+
+    func showReaderTab() {
+        fallbackBehavior()
+    }
+
+    func switchToDiscover() {
+        fallbackBehavior()
+    }
+
+    func switchToSavedPosts() {
+        fallbackBehavior()
+    }
+
+    func resetReaderDiscoverNudgeFlow() {
+        fallbackBehavior()
+    }
+
+    func resetReaderTab() {
+        fallbackBehavior()
+    }
+
+    func navigateToReaderSearch() {
+        fallbackBehavior()
+    }
+
+    func switchToTopic(where predicate: (ReaderAbstractTopic) -> Bool) {
+        fallbackBehavior()
+    }
+
+    func switchToMyLikes() {
+        fallbackBehavior()
+    }
+
+    func switchToFollowedSites() {
+        fallbackBehavior()
+    }
+
+    func navigateToReaderSite(_ topic: ReaderSiteTopic) {
+        fallbackBehavior()
+    }
+
+    func navigateToReaderTag(_ topic: ReaderTagTopic) {
+        fallbackBehavior()
+    }
+
+    func navigateToReader(_ pushControlller: UIViewController?) {
+        fallbackBehavior()
+    }
+
+    func showReaderTab(forPost: NSNumber!, onBlog: NSNumber!) {
+        fallbackBehavior()
+    }
+
+    // MARK: My Site
+
+    var mySitesCoordinator: MySitesCoordinator! {
```

Comment:
As I understand it's only needed because `WBTabBarController` does not have `nullability` identifiers? If yes, then I agree that `WPTabBarController` improvements are outside the scope.

```suggestion
    var mySitesCoordinator: MySitesCoordinator {
```

Reply:
> As I understand it's only needed because WBTabBarController does not have nullability identifiers?

That's correct, yes. After some thought, though, I believe it's worth it to add nullability identifiers to `WBTabBarController`. Adding nullability identifiers also allows us to include variables like `readerCoordinator` in `RootViewPresenter`. (Ref: https://github.com/wordpress-mobile/WordPress-iOS/pull/19817#discussion_r1058950919) 

Updated in fb960a3a0d8fb5d5390ee2911d8eb9425d1c8a45, f03e6c240341e14b357c14414fb8de8cf0f6781a, and 9b95aa5bb01290a9ebe39532ed52bfc27962c3a0 👍

Reply:
Thanks for including the nullability identifiers, I'll check the changes!